src/calculateRentalCost.js

```diff
@@ -20,7 +20,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const fullCost = 40;
+  const threeDaysDiscount = 20;
+  const sevenDaysDiscount = 50;
+
+  if (days < 3) {
+    return days * fullCost;
+  } else if (days >= 3 && days < 7) {
+    return days * fullCost - threeDaysDiscount;
+  } return days * fullCost - sevenDaysDiscount;
```

Comment:
Итоговый return на новой строке с отступом
```suggestion
  } 
  
return days * fullCost - sevenDaysDiscount;
```